assets/javascript/game.js

```diff
@@ -0,0 +1,106 @@
+// Crystal Collector Game Homework Pseudocode
+
+// 1.	To start the game, clicks on a link to go to the URL for the game.
+// 2.	H1 “CrystalsCollector!”
+// 3.	Paragraph text “You will be given a random number at the start of the game.”
```

Comment:
within your pseudo code, try to add your code within those lines explaining what you're doing